plugins/ports-plugin/package.json

```diff
@@ -16,6 +16,7 @@
     "@eclipse-che/plugin": "latest",
     "@theia/plugin": "latest",
     "@theia/plugin-packager": "latest",
+    "@types/jest": "24.0.3",
```

Comment:
We are using "jest": "22.1.4" for the factory-plugin.  
https://github.com/eclipse/che-theia/blob/master/plugins/factory-plugin/package.json#L17
Let's use the same version.